### ISSUE TITLE
where to go from here?

### ISSUE THREAD

**Issue by cahmer** [moritztng/hacker-league#4]
Ok, now we need a plan what to integrate, I think. What would be the most important things?
- texture support for playfield, walls, air, car etc.
- simple 3d model support for car and ball?
- scoreboard

later:
multiplayer / bot support
gfx/ sound etc.

Lets discuss and everybody should participate...

**Comment by moritztng** [comment]
Hacker League should be very simple and efficient.
So before we do any graphics, we should have very efficient physics (also for training AI) and robust multiplayer.
Right now, I'm cleaning up the code and simplifying a lot. I guess next is multiplayer and making physics even more efficient
I'm not sure whether we need textures at this stage.
Since the hitbox is a box anyways, 3d models are also not really necessary right now.
But clearly we need a very simple scoreboard, once we have multiplayer.

Great idea to discuss it here. Everyone is welcome!

**Comment by cahmer** [comment]
Ok you're right, that mechanics have to be there, first.

We need 1) jump 2) later a fly mode with boost :-)
3) Scoreboard

Then we get a bit further. If you need a graphical proposal for the scoreboard / user interface, I could heck something out :-)

**Comment by moritztng** [comment]
Yes jump will also be important! Graphical proposal sounds nice. And we need some creativity in community building :)
